app/src/main/java/com/appunite/loudius/analytics/events/ReviewersEvents.kt

```diff
@@ -44,45 +44,45 @@ object NotifyFailureEvent : ReviewersEvent {
     )
 }
 
-object RefreshDataEvent : ReviewersEvent {
+object RefreshReviewersEvent : ReviewersEvent {
```

Comment:
**out-of-scope**: what do you think about grouping events in some object:

```kotlin
object ReviewersEvents {
  object Refresh : ReviewersEvent {
    //
  }
 object RefreshSuccess : ReviewersEvent {
   //
 }
}
```

you can consider this in the future

Reply:
Ok, I added task.